systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -304,23 +301,16 @@ void testCruiseControlIntraBrokerBalancingWithoutSpecifyingJBODStorage(Extension
         final TestStorage testStorage = new TestStorage(extensionContext);
 
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build());
-        resourceManager.createResource(extensionContext, KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName())
+        resourceManager.createResource(extensionContext, false, KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName())
             .editOrNewSpec()
                 .withRebalanceDisk(true)
             .endSpec()
             .build());
 
-        final String coPodName = KubeClusterResource.kubeClient(INFRA_NAMESPACE).getClusterOperatorPodName();
-        final String coLog = StUtils.getLogFromPodByTime(INFRA_NAMESPACE, coPodName, STRIMZI_DEPLOYMENT_NAME, "60s");
-
-        // CO logs must contain warning message
-        assertThat(coLog, CoreMatchers.containsString("Intra-broker balancing only applies to Kafka deployments that use JBOD storage with multiple disks."));
+        KafkaRebalanceUtils.waitForKafkaRebalanceCustomResourceState(testStorage.getNamespaceName(), testStorage.getClusterName(), KafkaRebalanceState.NotReady);
 
-        KafkaRebalanceUtils.waitForKafkaRebalanceCustomResourceState(testStorage.getNamespaceName(), testStorage.getClusterName(), KafkaRebalanceState.ProposalReady);
-        // Strimzi will force an inter-broker balance when rebalanceDisk=`true` and a JBOD configuration is not found.
-        // The provision status should be "RIGHT_SIZED" because Strimzi is preforming an inter-broker balance here and the
-        // cluster should not be under or over provisioned by this rebalance.
+        // The intra-broker rebalance will fail for Kafka clusters with a non-JBOD configuration.
         KafkaRebalanceStatus kafkaRebalanceStatus = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(testStorage.getNamespaceName()).withName(testStorage.getClusterName()).get().getStatus();
-        assertThat(kafkaRebalanceStatus.getOptimizationResult().get("provisionStatus").toString(), containsString("RIGHT_SIZED"));
+        assertThat(kafkaRebalanceStatus.getConditions().get(0).getReason(), is("IllegalArgumentException"));
```

Comment:
I guess this now uses the old exception and needs to be updated?

Reply:
:man_facepalming: